client/app/cloud/project/add/cloud-project-add.controller.js

```diff
@@ -49,7 +49,7 @@ angular.module("managerApp").controller("CloudProjectAddCtrl",
             self.loaders.creating = true;
 
             // If contracts: accept them
-            if (self.model.contractsAccepted && self.data.agreements.length) {
+            //if (self.model.contractsAccepted && self.data.agreements.length) {
```

Comment:
- [ ] Check that contracts are accepted

Reply:
Nice catch.  I don't know why I commented this